nautobot/extras/tests/test_views.py

```diff
@@ -815,7 +850,6 @@ def test_get_object_dynamic_groups_with_permission(self):
         self.assertIn("DG 3", response_body, msg=response_body)
 
     def test_get_object_dynamic_groups_with_constrained_permission(self):
```

Comment:
What is the difference between this and the `test_get_object_with_constrained_permission` above?

Reply:
This is `/dcim/devices/<pk>/dynamic-groups/`, i.e. the "Dynamic Groups" tab of the Device detail view.

The test above is `/extras/dynamic-groups/<pk>`, i.e. the Dynamic Group detail view (which includes the "Members" tab, listing Devices in this case).

Two sides of the same coin.

Reply:
Got it. Thanks!